### PR TITLE
update of breaking change paragraphs

### PR DESCRIPTION
**Pull request by hdsheena**
I did my best :) Thanks @balloob for the coaching!

**Description:**
Just updating the breaking changes for the release notes.

## Checklist:

- [ RC ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x ] The documentation follows the [standards][standards].  (hope so!)

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_posts/2019-03-13-release-89.markdown [hunk 1]
```diff
@@ -52,18 +62,18 @@ Experiencing issues introduced by this release? Please report them in our [issue
...
+- Add yeelight flow action support - You can now specifiy an action when calling the yeelight_start_flow service on your yeelights. The __default action will be 'recover'__ if you do not specify something else. This action will be taken after the flow stops.  ([@zewelor] - [#21195]) ([light.yeelight docs]) (breaking change)
+- Prevent partial custom component overlays  Custom platforms that override a built-in platform that has a component, should now also include the component in the custom_components/ folder. So if you want to override `hue/light.py` with a custom version, __you will also need to provide/copy over `hue/__init__.py`.__ ([@balloob] - [#21070]) (breaking change)
```
Comment (balloob) [inline source/_posts/2019-03-13-release-89.markdown]: You can drop the commit message in a lot of these cases. So for example for the start of this line, drop "Prevent partial custom component overlays"

Comment (balloob) [reply source/_posts/2019-03-13-release-89.markdown]: Also, this is a very noteworthy breaking change 😬 . We should also link for more info to https://developers.home-assistant.io/blog/2019/02/19/the-great-migration.html

**Comment by balloob** [comment]
🎉 Awesome !

**Comment by speedmann** [comment]
Thank you very very much for this useful improvement.
Has this to be done by hand every time a release is made?
If yes, might it be possible for me to assist with this in the future?
After i was shitty last time i want to help improve more

**Comment by balloob** [comment]
Yeah, it's still manual work. When a beta is started, we generate a list of all PRs with breaking changes, but those are just titles. Each PR contains the breaking change and those need to be copied in + tweaked if they can be more helpful.

**Pull request merged**